core/cfgwatcher.go

```diff
@@ -0,0 +1,53 @@
+package core
+
+import (
+	"log"
+
+	"github.com/fsnotify/fsnotify"
+)
+
+type configWatchdog struct {
```

Comment:
I might suggest to rename this (and the file) to something more general instead of "config*", like maybe `dirWatchdog`/`dirwatcher.go`? -- the functionality implemented _could_ be used to watch any directory.

Reply:
yeap, sounds reasonable